platform-sdk/swirlds-merkledb/src/main/java/com/swirlds/merkledb/collections/LongListHeap.java

```diff
@@ -81,8 +80,8 @@ public LongListHeap(final int numLongsPerChunk) {
      * @param file the file to read from
      * @throws IOException If there was a problem reading the file
      */
-    public LongListHeap(final Path file, final Configuration configuration) throws IOException {
```

Comment:
cc @artemananiev

Reply:
I see two ways to deal with it:

1. Leave `configuration` in AbstractLongList. When a subclass instance is created, provide a config, even if it isn't used in this subclass
2. Remove `configuration` from AbstractLongList. Use it only in subclasses that really need a config

Reply:
> 2. Remove configuration from AbstractLongList. Use it only in subclasses that really need a config

I need the config only in `LongListDisk` but adding it only there is not an option since its constructor invokes `super()` which itself calls the overridden methods that need the config to be present. So I will go with the first option and revert it to the way it was i.e. for the other subclasses `AbstractLongList` I'll provide the config even though it's not used in them.

@poulok fyi, since it related to https://github.com/hashgraph/hedera-services/pull/14695#issuecomment-2465588945.